superset/sql_validators/presto_db.py

```diff
@@ -107,12 +112,16 @@ def validate_statement(
                     "The pyhive presto client did not report an error message"
                 ) from db_error
             if "errorLocation" not in error_args:
-                raise PrestoSQLValidationError(
-                    "The pyhive presto client did not report an error location"
-                ) from db_error
+                # Pylint is confused about the type of error_args, despite the hints
+                # and checks above.
+                # pylint: disable=invalid-sequence-index
+                message = error_args["message"] + "\n(Error location unknown)"
+                # If we have a message but no error location, return the message and
+                # set the location as the beginning.
+                return SQLValidationAnnotation(
+                    message=message, line_number=1, start_column=1, end_column=1
```

Comment:
Is this 1-indexed or 0-indexed? (I assume 1 since it's Presto, just checking.)

Reply:
1-indexed like you assumed

Reply:
1-indexed like you assumed